src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+          >
+        </label>
+
+        <label class="form_field">
+          Name:
+          <input type="text" name="first_name">
```

Comment:
```suggestion
          <input type="text" name="firstName">
```
[checklist p.4](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) use camelCase